BikeData/src/main/scala/by/itechart/internship/dao/DatabaseReader.scala

```diff
@@ -0,0 +1,15 @@
+package by.itechart.internship.dao
+
+import by.itechart.internship.entities.{TripInfo, TripInfoView}
+import by.itechart.internship.logic.MyPostgresDriver.api._
+
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.concurrent._
+
+object DatabaseReader {
+  def writer(): Future[List[TripInfo]] = {
+    val db = Database.forConfig("database")
```

Comment:
What if I want to use two databases in my app?
Try to pass DB into DAO somehow instead of Letting DAO know which DB to use.
For example, keep DAO as "object" and pass DB to method via implicit param (not the best option)
Better - change DAO to "class" and inject DB with some default value into constructor

Discussion:
https://stackoverflow.com/questions/40141762/play-2-5-dependency-injection-for-play-slick-database-configuration